zulipterminal/model.py

```diff
@@ -645,6 +645,13 @@ def toggle_stream_pinned_status(self, stream_id: int) -> bool:
         response = self.client.update_subscription_settings(request)
         return response['result'] == 'success'
 
+    def stream_ids_from_name(self, stream_name: str) -> int:
+        for stream_id, stream in self.stream_dict.items():
+            if stream['name'] == stream_name:
+                return stream_id
+
+        raise RuntimeError("Invalid stream name specified.")
```

Comment:
I'm using this just to deal with the mypy error of the method not having a return value. Do I return something like `''` instead?

Reply:
Well, we could do various things in case of error (empty string, `None`, custom exception, general exception), as long as we "handle" them somehow. mypy only knows return types, so exceptions act as a fix for this at some level from its perspective - and not always in a wrong way. As I said, having this exception is fine as long as we expect all stream names to be valid and otherwise it being an internal error - so here we don't expect to handle it. Later we might change it to be something that callers are expected to handle.

Reply:
Okay, this should be fine for now because the recipient validity check ensures that stream names are always valid.

Reply:
This was merged via the commit in #772, so this should be good for now.